src/vs/workbench/services/workingCopy/common/workingCopyFileService.ts

```diff
@@ -31,6 +31,13 @@ export interface SourceTargetPair {
 	readonly target: URI
 }
 
+export interface SourceTargetPairWithOptions extends SourceTargetPair {
+	options?: {
```

Comment:
This could get some JDoc comment what the purpose and impact of these options is.